diff '--color=auto' -c org/801-man-pages diff/801-man-pages
*** org/801-man-pages	2024-03-01 16:58:05.650605966 -0700
--- diff/801-man-pages	2024-03-01 16:58:24.699775043 -0700
***************
*** 48,54 ****
  ln -sv usr/bin  $PKG_DEST
  ln -sv usr/lib  $PKG_DEST
  make -j1 DESTDIR=$PKG_DEST prefix=/usr install
! rm -fv $PKG_DEST/usr/share/man/{man3/getspnam.3,man5/passwd.5}
  for dir in $PKG_DEST/usr/share/man/man{1..8} \
             $PKG_DEST/usr/share/bash-completion{/completions,} \
             $PKG_DEST/usr/share/{doc,info,man,} \
--- 48,54 ----
  ln -sv usr/bin  $PKG_DEST
  ln -sv usr/lib  $PKG_DEST
  make -j1 DESTDIR=$PKG_DEST prefix=/usr install
! #rm -fv $PKG_DEST/usr/share/man/{man3/getspnam.3,man5/passwd.5}
  for dir in $PKG_DEST/usr/share/man/man{1..8} \
             $PKG_DEST/usr/share/bash-completion{/completions,} \
             $PKG_DEST/usr/share/{doc,info,man,} \
diff '--color=auto' -c org/803-glibc diff/803-glibc
*** org/803-glibc	2024-03-01 16:58:05.650605966 -0700
--- diff/803-glibc	2024-03-01 16:58:24.700775052 -0700
***************
*** 68,73 ****
--- 68,80 ----
  mkdir -pv $PKG_DEST/var/cache/nscd
  install -v -Dm644 ../nscd/nscd.tmpfiles $PKG_DEST/usr/lib/tmpfiles.d/nscd.conf
  install -v -Dm644 ../nscd/nscd.service $PKG_DEST/usr/lib/systemd/system/nscd.service
+ 
+ cp -a /etc/localtime $PKG_DEST/etc
+ cp -a /etc/nsswitch.conf $PKG_DEST/etc
+ cp -a /etc/ld.so.conf $PKG_DEST/etc
+ cp -a /lib64/* $PKG_DEST/lib64
+ cp -a /usr/lib/locale $PKG_DEST/usr/lib
+ 
  for dir in $PKG_DEST/usr/share/man/man{1..8} \
             $PKG_DEST/usr/share/bash-completion{/completions,} \
             $PKG_DEST/usr/share/{doc,info,man,} \
diff '--color=auto' -c org/824-shadow diff/824-shadow
*** org/824-shadow	2024-03-01 16:58:05.655606010 -0700
--- diff/824-shadow	2024-03-01 16:58:24.703775079 -0700
***************
*** 62,69 ****
  ln -sv usr/lib  $PKG_DEST
  make -j1 DESTDIR=$PKG_DEST exec_prefix=/usr install
  make -j1 DESTDIR=$PKG_DEST -C man install-man
! rm -fv $PKG_DEST/usr/share/man/man8/nologin.8
! rm -fv $PKG_DEST/sbin/nologin
  for dir in $PKG_DEST/usr/share/man/man{1..8} \
             $PKG_DEST/usr/share/bash-completion{/completions,} \
             $PKG_DEST/usr/share/{doc,info,man,} \
--- 62,74 ----
  ln -sv usr/lib  $PKG_DEST
  make -j1 DESTDIR=$PKG_DEST exec_prefix=/usr install
  make -j1 DESTDIR=$PKG_DEST -C man install-man
! 
! cp -a /etc/default $PKG_DEST/etc/
! cp -a /etc/gshadow $PKG_DEST/etc
! cp -a /etc/shadow $PKG_DEST/etc
! 
! #rm -fv $PKG_DEST/usr/share/man/man8/nologin.8
! #rm -fv $PKG_DEST/sbin/nologin
  for dir in $PKG_DEST/usr/share/man/man{1..8} \
             $PKG_DEST/usr/share/bash-completion{/completions,} \
             $PKG_DEST/usr/share/{doc,info,man,} \
diff '--color=auto' -c org/825-gcc diff/825-gcc
*** org/825-gcc	2024-03-01 16:58:05.655606010 -0700
--- diff/825-gcc	2024-03-01 16:58:24.703775079 -0700
***************
*** 78,83 ****
--- 78,86 ----
          $PKG_DEST/usr/lib/bfd-plugins/
  mkdir -pv $PKG_DEST/usr/share/gdb/auto-load/usr/lib
  mv -v $PKG_DEST/usr/lib/*gdb.py $PKG_DEST/usr/share/gdb/auto-load/usr/lib
+ 
+ cp -a /usr/bin/cc $PKG_DEST/usr/bin
+ 
  for dir in $PKG_DEST/usr/share/man/man{1..8} \
             $PKG_DEST/usr/share/bash-completion{/completions,} \
             $PKG_DEST/usr/share/{doc,info,man,} \
diff '--color=auto' -c org/833-bash diff/833-bash
*** org/833-bash	2024-03-01 16:58:05.657606028 -0700
--- diff/833-bash	2024-03-01 16:58:24.704775088 -0700
***************
*** 60,65 ****
--- 60,68 ----
  ln -sv usr/bin  $PKG_DEST
  ln -sv usr/lib  $PKG_DEST
  make -j1 DESTDIR=$PKG_DEST install
+ 
+ cp -a /usr/bin/sh $PKG_DEST/usr/bin
+ 
  for dir in $PKG_DEST/usr/share/man/man{1..8} \
             $PKG_DEST/usr/share/bash-completion{/completions,} \
             $PKG_DEST/usr/share/{doc,info,man,} \
diff '--color=auto' -c org/849-Python diff/849-Python
*** org/849-Python	2024-03-01 16:58:05.661606064 -0700
--- diff/849-Python	2024-03-01 16:58:24.707775115 -0700
***************
*** 53,58 ****
--- 53,71 ----
  ln -sv usr/bin  $PKG_DEST
  ln -sv usr/lib  $PKG_DEST
  make -j1 DESTDIR=$PKG_DEST install
+ 
+ cp -a /usr/bin/pip3 $PKG_DEST/usr/bin
+ cp -a /usr/bin/pip3.11 $PKG_DEST/usr/bin
+ 
+ mkdir -pv $PKG_DEST/usr/lib/python3.11/site-packages/
+ cp -a /usr/lib/python3.11/site-packages/_distutils_hack $PKG_DEST/usr/lib/python3.11/site-packages/
+ cp -a /usr/lib/python3.11/site-packages/distutils-precedence.pth $PKG_DEST/usr/lib/python3.11/site-packages/
+ cp -a /usr/lib/python3.11/site-packages/pip $PKG_DEST/usr/lib/python3.11/site-packages/
+ cp -a /usr/lib/python3.11/site-packages/pip-23.1.2.dist-info $PKG_DEST/usr/lib/python3.11/site-packages/
+ cp -a /usr/lib/python3.11/site-packages/setuptools $PKG_DEST/usr/lib/python3.11/site-packages/
+ cp -a /usr/lib/python3.11/site-packages/setuptools-65.5.0.dist-info $PKG_DEST/usr/lib/python3.11/site-packages/
+ cp -a /usr/lib/python3.11/site-packages/pkg_resources $PKG_DEST/usr/lib/python3.11/site-packages/
+ 
  cat > $PKG_DEST/etc/pip.conf << EOF
  [global]
  root-user-action = ignore
diff '--color=auto' -c org/850-flit-core diff/850-flit-core
*** org/850-flit-core	2024-03-01 16:58:05.661606064 -0700
--- diff/850-flit-core	2024-03-01 16:58:24.707775115 -0700
***************
*** 48,53 ****
--- 48,58 ----
  ln -sv usr/lib  $PKG_DEST
  pip3 wheel -w dist --no-build-isolation --no-deps $PWD
  pip3 install --no-index --no-user --find-links dist flit_core
+ 
+ mkdir -pv $PKG_DEST/usr/lib/python3.11/site-packages/
+ cp -a /usr/lib/python3.11/site-packages/flit_core $PKG_DEST/usr/lib/python3.11/site-packages/
+ cp -a /usr/lib/python3.11/site-packages/flit_core-3.9.0.dist-info $PKG_DEST/usr/lib/python3.11/site-packages/
+ 
  for dir in $PKG_DEST/usr/share/man/man{1..8} \
             $PKG_DEST/usr/share/bash-completion{/completions,} \
             $PKG_DEST/usr/share/{doc,info,man,} \
diff '--color=auto' -c org/851-wheel diff/851-wheel
*** org/851-wheel	2024-03-01 16:58:05.661606064 -0700
--- diff/851-wheel	2024-03-01 16:58:24.707775115 -0700
***************
*** 48,53 ****
--- 48,60 ----
  ln -sv usr/bin  $PKG_DEST
  ln -sv usr/lib  $PKG_DEST
  pip3 install --no-index --find-links=dist wheel
+ 
+ cp -a /usr/bin/wheel $PKG_DEST/usr/bin
+ 
+ mkdir -pv $PKG_DEST/usr/lib/python3.11/site-packages/
+ cp -a /usr/lib/python3.11/site-packages/wheel $PKG_DEST/usr/lib/python3.11/site-packages/
+ cp -a /usr/lib/python3.11/site-packages/wheel-0.41.1.dist-info $PKG_DEST/usr/lib/python3.11/site-packages/
+ 
  for dir in $PKG_DEST/usr/share/man/man{1..8} \
             $PKG_DEST/usr/share/bash-completion{/completions,} \
             $PKG_DEST/usr/share/{doc,info,man,} \
diff '--color=auto' -c org/853-meson diff/853-meson
*** org/853-meson	2024-03-01 16:58:05.662606072 -0700
--- diff/853-meson	2024-03-01 16:58:24.708775124 -0700
***************
*** 50,55 ****
--- 50,64 ----
  pip3 install --no-index --find-links dist meson
  install -vDm644 data/shell-completions/bash/meson $PKG_DEST/usr/share/bash-completion/completions/meson
  install -vDm644 data/shell-completions/zsh/_meson $PKG_DEST/usr/share/zsh/site-functions/_meson
+ 
+ cp -a /usr/bin/meson $PKG_DEST/usr/bin
+ mkdir -pv $PKG_DEST/usr/lib/python3.11/site-packages
+ cp -a /usr/lib/python3.11/site-packages/meson-1.2.1.dist-info $PKG_DEST/usr/lib/python3.11/site-packages
+ cp -a /usr/lib/python3.11/site-packages/mesonbuild $PKG_DEST/usr/lib/python3.11/site-packages
+ cp -a /usr/share/man/man1/meson.1 $PKG_DEST/usr/share/man/man1
+ mkdir -pv $PKG_DEST/usr/share/polkit-1/actions
+ cp -a /usr/share/polkit-1/actions/com.mesonbuild.install.policy $PKG_DEST/usr/share/polkit-1/actions
+ 
  for dir in $PKG_DEST/usr/share/man/man{1..8} \
             $PKG_DEST/usr/share/bash-completion{/completions,} \
             $PKG_DEST/usr/share/{doc,info,man,} \
diff '--color=auto' -c org/869-vim diff/869-vim
*** org/869-vim	2024-03-01 16:58:05.666606108 -0700
--- diff/869-vim	2024-03-01 16:58:24.710775141 -0700
***************
*** 57,62 ****
--- 57,65 ----
      ln -sv vim.1 $(dirname $L)/vi.1
  done
  ln -sv ../vim/vim90/doc $PKG_DEST/usr/share/doc/vim-9.0.1677
+ 
+ cp -a /etc/vimrc $PKG_DEST/etc
+ 
  for dir in $PKG_DEST/usr/share/man/man{1..8} \
             $PKG_DEST/usr/share/bash-completion{/completions,} \
             $PKG_DEST/usr/share/{doc,info,man,} \
diff '--color=auto' -c org/870-markupsafe diff/870-markupsafe
*** org/870-markupsafe	2024-03-01 16:58:05.666606108 -0700
--- diff/870-markupsafe	2024-03-01 16:58:24.710775141 -0700
***************
*** 48,53 ****
--- 48,58 ----
  ln -sv usr/bin  $PKG_DEST
  ln -sv usr/lib  $PKG_DEST
  pip3 install --no-index --no-user --find-links dist Markupsafe
+ 
+ mkdir -pv $PKG_DEST/usr/lib/python3.11/site-packages/
+ cp -a /usr/lib/python3.11/site-packages/markupsafe $PKG_DEST/usr/lib/python3.11/site-packages/
+ cp -a /usr/lib/python3.11/site-packages/MarkupSafe-2.1.3.dist-info $PKG_DEST/usr/lib/python3.11/site-packages/
+ 
  for dir in $PKG_DEST/usr/share/man/man{1..8} \
             $PKG_DEST/usr/share/bash-completion{/completions,} \
             $PKG_DEST/usr/share/{doc,info,man,} \
diff '--color=auto' -c org/871-jinja2 diff/871-jinja2
*** org/871-jinja2	2024-03-01 16:58:05.666606108 -0700
--- diff/871-jinja2	2024-03-01 16:58:24.711775150 -0700
***************
*** 48,53 ****
--- 48,58 ----
  ln -sv usr/lib  $PKG_DEST
  pip3 wheel -w dist --no-build-isolation --no-deps $PWD
  pip3 install --no-index --no-user --find-links dist Jinja2
+ 
+ mkdir -pv $PKG_DEST/usr/lib/python3.11/site-packages/
+ cp -a /usr/lib/python3.11/site-packages/jinja2 $PKG_DEST/usr/lib/python3.11/site-packages/
+ cp -a /usr/lib/python3.11/site-packages/Jinja2-3.1.2.dist-info $PKG_DEST/usr/lib/python3.11/site-packages/
+ 
  for dir in $PKG_DEST/usr/share/man/man{1..8} \
             $PKG_DEST/usr/share/bash-completion{/completions,} \
             $PKG_DEST/usr/share/{doc,info,man,} \
diff '--color=auto' -c org/876-util-linux diff/876-util-linux
*** org/876-util-linux	2024-03-01 16:58:05.667606117 -0700
--- diff/876-util-linux	2024-03-01 16:58:24.711775150 -0700
***************
*** 66,71 ****
--- 66,75 ----
  ln -sv usr/bin  $PKG_DEST
  ln -sv usr/lib  $PKG_DEST
  make -j1 DESTDIR=$PKG_DEST install
+ 
+ mkdir -pv $PKG_DEST/var/lib
+ cp -a /var/lib/hwclock $PKG_DEST/var/lib
+ 
  for dir in $PKG_DEST/usr/share/man/man{1..8} \
             $PKG_DEST/usr/share/bash-completion{/completions,} \
             $PKG_DEST/usr/share/{doc,info,man,} \
