--- org/503-linux-headers	2024-03-02 16:29:23.590109918 -0700
+++ diff/503-linux-headers	2024-03-02 16:29:40.515299400 -0700
@@ -16,6 +16,8 @@
 PKG_DEST=${SRC_DIR}/503-linux-headers
 
 PACKAGE=linux-6.4.12.tar.xz
+source ${ROOT}${SCRIPT_ROOT}/packInstall.sh
+export -f packInstall
 cd $SRC_DIR
 PKGDIR=$(tar -tf $PACKAGE | head -n1 | sed 's@^./@@;s@/.*@@')
 export PKGDIR VERSION PKG_DEST
@@ -38,7 +40,11 @@
 make headers
 find usr/include -type f ! -name '*.h' -delete
 
-cp -rv usr/include $LFS/usr
+mkdir -pv $PKG_DEST/usr
+cp -rv usr/include $PKG_DEST/usr
+
+packInstall
+
 echo -e "\n\nTotalseconds: $SECONDS\n"
 
 # End of LFS book script
